public/js/like.js

```diff
@@ -17,7 +17,7 @@ $(document).ready(function() {
 
 			like_count = Number(like_count) + 1;
 
-			$(this).text(like_count);
+			$(this).text( " " + like_count);
```

Comment:
Mixed double and single quotes.<br>Identifier 'like_count' is not in camel case.